doc/tutorials/objdetect/aruco_detection/aruco_detection.markdown

```diff
@@ -0,0 +1,802 @@
+coefficients: a vector of 5 or more elements that models the distortion produced by your camera.
+
+When you estimate the pose with ArUco markers, you can estimate the pose of each marker individually.
+If you want to estimate one pose from a set of markers, use ArUco Boards (see the **Detection of ArUco
+Boards** tutorial). Using ArUco boards instead of single markers allows some markers to be occluded.
+
+The camera pose relative to the marker is a 3d transformation from the marker coordinate system to the
+camera coordinate system. It is specified by rotation and translation vectors (see `cv::solvePnP()` function for more
+information).
+
+@code{.cpp}
+cv::Mat cameraMatrix, distCoeffs;
+// You can read camera parameters from tutorial_camera_params.yml
+readCameraParameters(cameraParamsFilename, cameraMatrix, distCoeffs); // This function is implemented in aruco_samples_utility.hpp
+
+std::vector<cv::Vec3d> rvecs, tvecs;
+
+// Set coordinate system
+cv::Mat objPoints(4, 1, CV_32FC3);
+...
+
+// Calculate pose for each marker
+for (int i = 0; i < nMarkers; i++) {
+    solvePnP(objPoints, corners.at(i), cameraMatrix, distCoeffs, rvecs.at(i), tvecs.at(i));
+}
+@endcode
+
+- The `markerCorners` parameter is the vector of marker corners returned by the `cv::aruco::ArucoDetector::detectMarkers()` function.
+- The second parameter is the size of the marker side in meters or in any other unit. Note that the
+translation vectors of the estimated poses will be in the same unit
+- `cameraMatrix` and `distCoeffs` are the camera calibration parameters that were created during the camera calibration process.
+- The output parameters `rvecs` and `tvecs` are the rotation and translation vectors respectively, for each of the markers
+in `markerCorners`.
+
+The marker coordinate system that is assumed by this function is placed in the center (by default) or
+in the top left corner of the marker with the Z axis pointing out, as in the following image.
+Axis-color correspondences are X: red, Y: green, Z: blue. Note the axis directions of the rotated markers in this image.
+
+![Image with axes drawn](images/singlemarkersaxes.jpg)
+
+OpenCV provides a function to draw the axis as in the image above, so pose estimation can be
+checked:
+
+@code{.cpp}
+inputImage.copyTo(outputImage);
+for (int i = 0; i < rvecs.size(); ++i) {
+    auto rvec = rvecs[i];
+    auto tvec = tvecs[i];
+    cv::drawFrameAxes(outputImage, cameraMatrix, distCoeffs, rvec, tvec, 0.1);
+}
+@endcode
+
+- `outputImage ` is the input/output image where the markers will be drawn (it will normally be the same image where the markers were detected).
+- `cameraMatrix` and `distCoeffs` are the camera calibration parameters.
+- `rvec` and `tvec` are the pose parameters for the marker whose axis is to be drawn.
+- The last parameter is the length of the axis, in the same unit as tvec (usually meters).
+
+A basic full example for pose estimation from single markers:
```

Comment:
Please rewrite the sentence.

Reply:
![image](https://github.com/opencv/opencv/assets/22337800/fc778fcc-fb1a-465f-bbc3-848108eb2481)

Reply:
![image](https://github.com/opencv/opencv/assets/22337800/0b26d369-20e4-45df-b3d0-57e31506b85e)

Reply:
![Uploading image.png…]()